designer/client/src/components/graph/utils/graphUtils.ts

```diff
@@ -121,3 +123,11 @@ function addEdge(process: Process, fromId: NodeId, toId: NodeId): Process {
         edges: process.edges.concat({ from: fromId, to: toId }),
     };
 }
+
+export const handleGraphEvent =
```

Comment:
it's only for cell view events.  i'd like to see in name when it's in separate module

Reply:
I made it more generic, so it will be possible to pass any View based type